v3/soda/mango/mango.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "MF17",
+    "vendorId": "0x1A54",

```

Comment:
wrong VID

Reply:
I have  update v3 vid and rm src v2 json

Reply:
My keyboard is already in mass production, I need this vid, can't change it. Please let me pass